docs/quickstart/setting_env_vars.adoc

```diff
@@ -0,0 +1,39 @@
+ifndef::imagesdir[:imagesdir: ../]
+=== Setting Environment Variables
+
+One of the more complicated things to setup are all of the settings that Terraform requires to get things up and running. To make this easier, two tasks have been defined that will generate either a PowerShell or Bash script which will allow the values to be set.
+
+Run the appropriate task for the output that is required:
+
+[cols="1a,1a",options="header"]
+|===
+| PowerShell | Bash
+|
+
+[source,powershell]
+----
+# Generate the powershell script file
+eirctl local:envfile:powershell
```

Comment:
first line of envfile_infra.ps1 is causing error when trying to set the env variables
```
(base) PS /Users/josecarrera/dev/ensono/stacks-azure-fabric> . ./local/envfile_infra.ps1
.: /Users/josecarrera/dev/ensono/stacks-azure-fabric/local/envfile_infra.ps1:1:3
Line |
   1 |  . (Split-Path -Parent -Path $PSScriptRoot)/credentials.ps1
     |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The term '/Users/josecarrera/dev/ensono/stacks-azure-fabric' is not recognized as a name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Reply:
I have fixed this